projects/ui/src/components/Analytics/Field/index.tsx

```diff
@@ -9,22 +9,34 @@ import Temperature from './Temperature';
 import Sown from './Sown';
 import TotalSowers from './TotalSowers';
 import RRoR from './RRoR';
-
-// const SLUGS = ['rror', 'weather', 'pods', 'podrate', 'sown', 'harvested', 'sowers'];
 import { FC } from '~/types';
 
+const SLUGS = [
+  'rror',
+  'weather',
+  'pods',
+  'podrate',
+  'sown',
+  'harvested',
+  'sowers',
+];
+
 const FieldAnalytics: FC<{}> = () => {
-  const [tab, handleChangeTab] = useTabs();
+  const [tab, handleChangeTab] = useTabs(SLUGS, 'field');
   return (
     <Card>
-      <Tabs value={tab} onChange={handleChangeTab} sx={{ px: 2, pt: 2, pb: 1.5 }}>
+      <Tabs
+        value={tab}
+        onChange={handleChangeTab}
+        sx={{ px: 2, pt: 2, pb: 1.5 }}
+      >
         <Tab label="RRoR" />
         <Tab label="Temperature" />
         <Tab label="Pods" />
         <Tab label="Pod Rate" />
         <Tab label="Sown" />
         <Tab label="Harvested" />
-        <Tab label="Total Sowers" />
+        <Tab label="TotalSowers" />
```

Comment:
See above comments, we'll want to keep this as "Total Sowers" for cosmetic & readability reasons. (The `label` is what's actually disabled for the user to click on)